usagov/usagov-2021
# USAGOV-1346 call center wait times

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1346

## Description
Added infrastructure for getting estimated wait time for call center to be displayed on /phone and /es/llamenos.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions
PRETESTING INSTRUCTIONS: Must add the following to /phone and /es/llamenos:
```
<div id="callCenterTime">
    &nbsp;
</div>
```

Then navigating to those pages should show the estimated wait time at the bottom of content.  If not, talk to me to work out why not.

### Requires New Config

### Requires New Content

### Validation Steps
- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
